examples/udpsrc_benchmark.rs

```diff
@@ -95,6 +95,20 @@ fn main() {
 
                 source
             }
+            "ts-tcpclientsrc" => {
```

Comment:
In a separate commit, this should probably be renamed from `udpsrc_benchmark` to `benchmark` in the end :)

Reply:
It's OK, No Problem.